contracts/token/Minter.sol

```diff
@@ -131,33 +131,33 @@ contract Minter is Manager, IMinter {
     }
 
     /**
-     * @notice Set maxInflation. Only callable by Controller owner
-     * @param _maxInflation New inflation cap as a percentage of total token supply
+     * @notice Set inflationCeiling. Only callable by Controller owner
+     * @param _inflationCeiling New inflation cap as a percentage of total token supply
      */
-    function setMaxInflation(uint256 _maxInflation) external onlyControllerOwner {
+    function setinflationCeiling(uint256 _inflationCeiling) external onlyControllerOwner {
         // Must be valid percentage
-        require(MathUtils.validPerc(_maxInflation), "_maxInflation is invalid percentage");
+        require(MathUtils.validPerc(_inflationCeiling), "_inflationCeiling is invalid percentage");
         // Inflation ceiling should be higher or equal to the floor
-        require(_maxInflation >= minInflation, "_maxInflation must be >= minInflation");
+        require(_inflationCeiling >= inflationFloor, "_inflationCeiling must be >= inflationFloor");
 
-        maxInflation = _maxInflation;
+        inflationCeiling = _inflationCeiling;
 
-        emit ParameterUpdate("maxInflation");
+        emit ParameterUpdate("inflationCeiling");
     }
 
     /**
-     * @notice Set minInflation. Only callable by Controller owner
-     * @param _minInflation New inflation floor as a percentage of total token supply
+     * @notice Set inflationFloor. Only callable by Controller owner
+     * @param _inflationFloor New inflation floor as a percentage of total token supply
      */
-    function setMinInflation(uint256 _minInflation) external onlyControllerOwner {
+    function setinflationFloor(uint256 _inflationFloor) external onlyControllerOwner {
```

Comment:
Probably mis-replaced the name here:
```suggestion
    function setInflationFloor(uint256 _inflationFloor) external onlyControllerOwner {
```

Will need to rename in other places like tests etc